ci_framework/roles/openshift_provisioner_node/meta/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+galaxy_info:
+  author: CI Framework
+  description: CI Framework Role -- openshift_provisioner_node
+  company: Red Hat
+  license: Apache-2.0
+  min_ansible_version: 2.14
+  namespace: edpm
```

Comment:
edpm namespace correct?

Reply:
since we're not really releasing any collection, we may use any namespace - that's mostly for ansible-galaxy actually. Might as well be `cifmw` in fact.. Maybe to consider IF we actually go on the "release collection" path (not planed so far)

Reply:
Thanks for the clarification and information. Let me leave it as is for now.

Reply:
OK, but if 'we may use any namespace' then why don't we just call this "cifmw_opn" or something (openshift provisioner node opn) it seems arbitrary to me that we use the namespace of another collection if there is no reason to do so.

Like if we _had_ to use edpm cos reasons then fine. if we can use whatever we want may as well make it more relevant